config/locales/en.yml

```diff
@@ -76,6 +76,10 @@ en:
       author: 'Author'
       description: 'Description'
       show_places: 'Show places in my area'
+      new_event: 'Dodaj nowe wydarzenie'
+      destroy: 'Usuń'
```

Comment:
You do duplication of translations and you add Polish translations to English file.